build/VSIX.targets

```diff
@@ -0,0 +1,67 @@
+<Project>
+  <PropertyGroup>
+    <PackageDependsOn>$(PackageDependsOn);GenerateVSIX</PackageDependsOn>
+    <NuGetCommandLineVersion>3.5.0</NuGetCommandLineVersion>
+    <VSIXName>Microsoft.VisualStudio.RazorExtension</VSIXName>
+    <VSIXProject>$(RepositoryRoot)tooling\$(VSIXName)\$(VSIXName).csproj</VSIXProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NuGet.CommandLine" Version="$(NuGetCommandLineVersion)" />
+  </ItemGroup>
+
+  <Target Name="GenerateVSIX" DependsOnTargets="LocateMSBuildExe;BuildVSIX" />
```

Comment:
All targets except this one should follow @natemcmaster's internal convention -- start name with an underscore. Nothing else can usefully be called except from within this file.